testsuite/tests/prometheus/apicast/test_metrics.py

```diff
@@ -58,28 +59,36 @@ def test_metrics_from_target_must_contains_apicast_metrics(expected_metric, metr
 
 # there is certain delay before all appears in Prometheus
 @backoff.on_predicate(backoff.fibo, lambda x: sorted(x.keys()) != STATUSES, max_tries=8, jitter=None)
-def apicast_status_metrics(prometheus):
+def apicast_status_metrics(prometheus, container):
     """Reliable gathering of prometheus metrics
 
     Metrics in prometheus appear with some delay, therefore retry is needed
     to ensure expected values are available"""
     hits = prometheus.get_metric("apicast_status")
 
     return {
-        status: int(hit["value"][1]) for status in STATUSES for hit in hits if int(hit["metric"]["status"]) == status}
+        status: int(hit["value"][1]) for status in STATUSES
+        for hit in hits if int(hit["metric"]["status"]) == status and
+        hit["metric"]["container"] == container
+    }
 
 
 @pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-5417")
-def test_apicast_status_metrics(client, prometheus):
+@pytest.mark.parametrize(("client", "apicast"), [("api_client", "apicast-staging"),
```

Comment:
`apicast` is called `container` elsewhere, it should be called everywhere same (and maybe in this context container seems bit better, however I can't say)

Reply:
Right, I was in impression that I used getfixturevalue and used the clients, but it's not the case and it's used only as container name, will fix it.